### PR TITLE
Document power up ability of pin 5

### PR DESCRIPTION
**Pull request by matthijskooijman**

### CODE REVIEW

**Comment by matthijskooijman** [comment]
For full disclosure, I added a link to the documentation to my own blog, since I think this is the most comprehensive discussion of this wakup ability that I'm aware of. If there's better references, feel free to update the link.

**Comment by Gadgetoid** [comment]
Thank you- I'd say an external documentation link should be a primary source- ie: the elinux wiki or Raspberry Pi's own documentation- however neither seem to mention this in more than passing (which you mention).

In this case it might be better to make contributions to the elinux wiki (https://elinux.org/Special:RequestAccount) or Raspberry Pi documentation itself (https://github.com/raspberrypi/documentation).

**Comment by seamusdemora** [comment]
@Gadgetoid : 
> Thank you- I'd say an external documentation link should be a primary source- ie: the elinux wiki or Raspberry Pi's own documentation- however neither seem to mention this in more than passing (which you mention).
> 
> In this case it might be better to make contributions to the elinux wiki (https://elinux.org/Special:RequestAccount) or Raspberry Pi documentation itself (https://github.com/raspberrypi/documentation). 

I see your point, but circumstances suggest that may result in quite a long wait: 

The elinux wiki page seems *out-of-date* with only [2 revisions in the past 6 years](https://elinux.org/index.php?title=RPi_Low-level_peripherals&action=history), and no new material since [`Model A+, B+ and B2`](https://elinux.org/RPi_Low-level_peripherals#Model_A.2B.2C_B.2B_and_B2).  Even the [Raspberry Pi Hub page](https://elinux.org/RPi_Hub) is years out-of-date. Is this considered to be a "reliable source"? 

Wrt The Foundation's documentation: Oddly, the only *concrete* documentation I can find is in their [`/boot/overlays/README` file](https://github.com/raspberrypi/firmware/blob/master/boot/overlays/README) under the overlay called [`gpio-shutdown`](https://github.com/seamusdemora/PiFormulae/blob/master/docs/gpio-shutdown_20210620.md). If that's not sufficient, then perhaps this pin (GPIO 3) may never be fully documented in `Pinout.xyz`?? There is an abundance of *anecdotal* documentation (RPi forums, independent blogs, Stack Exchange, etc) to support this function for GPIO 3, but given The Foundation's long-standing omission from their documentation, it seems doubtful it will ever be included.  

In summary, and FWIW, I feel that @matthijskooijman's proposal would improve the utility of `Pinout.xyz`.

**Comment by matthijskooijman** [comment]
> In this case it might be better to make contributions to the elinux wiki (https://elinux.org/Special:RequestAccount) or Raspberry Pi documentation itself (https://github.com/raspberrypi/documentation).

I'm not willing to free up enough time to get that documentation up-to-date in this area, sorry.

> Oddly, the only concrete documentation I can find is in their /boot/overlays/README file under the overlay called gpio-shutdown. 

FWIW, that was also written by me when I submitted that overlay when I first wrote the linked blogpost.

If it helps: I have been taking the effort of keeping my blogpost up-to-date with newer developments, based on comments and my own discoveries (i.e. for newer hardware, or when relevant documentation or discussions show up).

**Comment by lurch** [comment]
Old issue, and I've never tested this myself, but I suspect that this wouldn't work on Pi 5, as the GPIO pins are connected via the RP1 chip and not directly to the SoC. However the Pi 5 has a dedicated power-button anyway (and matching solderable-holes for an external button) :slightly_smiling_face:
